packages/jaspr_builder/lib/src/imports/analyzing_builder.dart

```diff
@@ -29,9 +29,11 @@ class ImportsModuleBuilder implements Builder {
       var outputId = buildStep.inputId.changeExtension('.imports.json');
       var partId = buildStep.inputId.changeExtension('.imports.dart');
 
-      var import = lib.libraryImports
+      // Get the library's compilation unit and use its imports instead
+      var compilationUnit = lib.definingCompilationUnit;
```

Comment:
Thanks for taking a look at the deprecation warnings, but these should not be updated and instead just ignored. Because e.g. the `compilationUnit.libraryImports` is only available from analyzer 6.10 but the current dependency constraint is set to `^6.5.0`.